restart_process/test/test-custom.sh

```diff
@@ -13,7 +13,7 @@ if [ $CI_EXIT -eq 0 ]; then
   exit 1
 fi
 
-cat tilt.log | grep -q "Are you there, pod?"
+grep -q "Are you there, pod?" tilt.log
```

Comment:
aw, kinda disappointed shellcheck is one of *those people*

Reply:
the shell input redirect fight reminds me of that old UX meme:
![1_pMk3h0dIYMb_I1iJCjriPQ](https://user-images.githubusercontent.com/278641/126541945-d93af09d-3409-47e8-9c3d-371324a2ba82.jpeg)